cfgov/jinja2/v1/youth_employment_success/index.html

```diff
@@ -1,62 +1,75 @@
-{% extends "base.html" %}
+{% extends 'layout-side-nav.html' %}
 
 {% block css -%}
   {{ super() }}
   <link rel="stylesheet" href="{{ static('apps/youth-employment-success/css/main.css') }}">
-{%- endblock css %}
+{%- endblock %}
 
 {% block title -%}
   Planning how you’ll get to work | {{ _('Consumer Financial Protection Bureau') }}
 {%- endblock %}
 
 {%- block desc -%}
-  This tool helps youth plan how to get to work, budget for transportation costs, and identify a backup plan. 
+  This tool helps youth plan how to get to work, budget for transportation costs, and identify a backup plan.
 {% endblock %}
 
+{% block content_main_modifiers -%}
+    {{ super() }} content__1-3 yes-transit-tool
+{%- endblock %}
 
-{% block content %}
+{% block pre_content %}
+    <div class="content_wrapper">
+        <nav class="breadcrumbs" aria-label="Breadcrumbs">
+            {{ svg_icon('left') }}
+            <a class="breadcrumbs_link" href="/consumer-tools/educator-tools/">
+              Educator Tools
+            </a>
+            <a class="breadcrumbs_link" href="/consumer-tools/educator-tools/resources-youth-employment-programs/">
+              Resources for Youth Employment Programs
```

Comment:
This doesn't appear to have saved much over having the template, but several pages use a block like this and if we refactor the base template they use, this block can go for all of them.

Reply:
Let me know if and when the base template is refactored